src/mumble/AudioOutput.cpp

```diff
@@ -496,19 +496,15 @@ bool AudioOutput::mix(void *outbuff, unsigned int nsamp) {
 
 			if (recorder) {
 				AudioOutputSpeech *aos = qobject_cast<AudioOutputSpeech *>(aop);
+				Q_ASSERT(aos != nullptr);
```

Comment:
Oh shit, how did I even miss this? I briefly looked at this wondering, but then went through the list in the description.

It seems I only looked at the change below, which is fine even without the assertion (dropping an additional if that is always true after the first one). I totally missed the assert then.

Yeah, removing the assert.